lib/rspec/mocks/space.rb

```diff
@@ -56,6 +56,8 @@ def raise_lifecycle_message
     # @private
     class Space
       attr_reader :proxies, :any_instance_recorders, :proxy_mutex, :any_instance_mutex
+      attr_accessor :temporarily_suppressing_verification
+      alias :temporarily_suppressing_verification? :temporarily_suppressing_verification
```

Comment:
I'm open to moving the state into configuration, but I was worried about the use of global state, and exposing this as a public api (I think it should be private)